tests/conftest.py

```diff
@@ -53,6 +53,7 @@ def provider_model_map() -> dict[ProviderName, str]:
         ProviderName.PORTKEY: "@first-integrati-d8a10f/gpt-4.1-mini",  # Owned by njbrake in portkey UI
         ProviderName.LLAMA: "Llama-4-Maverick-17B-128E-Instruct-FP8",
         ProviderName.AZURE: "openai/gpt-4.1-nano",
+        ProviderName.AZUREOPENAI: "azure/<your_deployment_name>",
```

Comment:
@njbrake / @besaleli not super critical but it feels annoying to have this separate dicts in the conftest. I don't know if it make sense to have like this values stored at the class level like `TEST_COMPLETION_MODEL` or something so when someone adds a new provider they can set that and if it's not set then we skip the test?